src/oic/utils/http_util.py

```diff
@@ -546,14 +546,15 @@ def wsgi_wrapper(environ, start_response, func, **kwargs):
 
 
 class CookieDealer(object):
-    def getServer(self):
+
+    @property
+    def srv(self):
         return self._srv
 
-    def setServer(self, server):
+    @srv.setter
+    def srv(self, server):
         self._srv = server
 
-    srv = property(getServer, setServer)
-
     def __init__(self, srv, ttl=5, secure=True, httponly=True):
         self.srv = None
```

Comment:
Why keep this when having the property above?

Reply:
True